lib/sync-util.js

```diff
@@ -1,11 +1,27 @@
 var crypto = require('crypto');
 var winston = require('winston');
+var asyncJson = require('async-json');
 var moment = require('moment');
 var assert = require('assert');
 
 var SYNC_LOGGER = 'SYNC';
 var loggers = {};
 
+var asyncGenerateHash = function(plainText, cb) {
+  if(plainText) {
+    console.log('generateHash - async mode');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)